components/atom/tooltip/src/AtomTooltip.js

```diff
@@ -0,0 +1,135 @@
+import {forwardRef, useCallback, useRef, useMemo} from 'react'
+import useControlledState from '@s-ui/react-hooks/lib/useControlledState'
+import loadable from '@loadable/component'
+import PropTypes from 'prop-types'
+import cx from 'classnames'
+import UAParser from 'ua-parser-js'
+import useIntersection from 'react-use/lib/useIntersection'
+
+import {
+  BASE_CLASS,
+  CLASS_INNER,
+  CLASS_ARROW,
+  PREFIX_PLACEMENT,
+  CLASS_TARGET,
+  COLORS,
+  PLACEMENTS,
+  DEFAULT_OFFSET,
+  TRIGGERS
+} from './config'
+
+import TooltipExtendChildren from './TooltipExtendChildren'
+
+const Tooltip = loadable(() => import('reactstrap/lib/Tooltip'), {ssr: true})
+
+const AtomTooltip = forwardRef(
+  (
+    {
+      children,
+      color,
+      content,
+      defaultIsVisible,
+      delay,
+      isArrowed = true,
+      isVisible,
```

Comment:
As you were commented in the SUI Components Weekly today, we would define the default value to `true`, right?

```suggestion
      isVisible = true,
```

Reply:
default naming:
- if bolean props (prefix is)
- semantic affirmative (visible – no hidden)

resulting attribute name: `isVisible`

the default value depends in the behaviour desired for the component